src/librustc_trans/save/dump_csv.rs

```diff
@@ -1118,10 +1178,11 @@ impl<'l, 'tcx, 'v> Visitor<'v> for DumpCsvVisitor<'l, 'tcx> {
     }
 
     fn visit_mac(&mut self, _: &ast::Mac) {
-        // Just stop, macros are poison to us.
+        // These shouldn't exist in the AST at this point, do nothing.
```

Comment:
Fair point. In the underlying default implementation it panics, this empty implementation overrides that. If we just want to panic we could remove the entire method and leave it to default behaviour.